Resolve `_FileToAnalyzeEntry` Error

When checking the output of the tool against versions of my own package, I'm met with many `[ERROR] { exception: Problem parsing _FileToAnalyzeEntry(filePath: /src/batch.dart, shownNames: [], hiddenNames: [], exportedBy: {firefuel.dart}): Bad state: Unable to find the context to /src/batch.dart, stacktrace: null }` errors.

## Reproduce
Run `dart-apitool diff --old pub://firefuel/0.3.1 --new pub://firefuel/0.3.2`

## Package Resources
- pub.dev: https://pub.dev/packages/firefuel
- github: https://github.com/SupposedlySam/firefuel/tree/main/packages/firefuel

## Discussion
Would you advise on whether this is an issue with my package or if it is a bug within the tool?

Thanks!

--- a/.fvm/fvm_config.json
+++ b/.fvm/fvm_config.json
@@ -1,4 +1,4 @@
 {
-  "flutterSdkVersion": "3.7.0",
+  "flutterSdkVersion": "3.7.3",
   "flavors": {}
 }

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 0.11.1
+- fixes parsing of local package references
+
 ## Version 0.11.0
 - replaces `--check-versions` with `--version-check-mode`
 - reduces the amount of directories to copy in a path-dependency-context

--- a/lib/src/analyze/package_api_analyzer.dart
+++ b/lib/src/analyze/package_api_analyzer.dart
@@ -409,7 +409,7 @@ class PackageApiAnalyzer {
       throw ArgumentError.value(libraryIdentifier, 'libraryIdentifier',
           'Looks like a package (starts with \'package:\' but doesn\'t contain \'/\'');
     }
-    return libraryIdentifier.substring(libraryIdentifier.indexOf('/'));
+    return libraryIdentifier.substring(libraryIdentifier.indexOf('/') + 1);
   }
 
   bool _isInternalRef(

